.github/workflows/test.yml

```
@@ -44,3 +44,21 @@ jobs:
         run: |
           make test-unit-cover
         if: env.GIT_DIFF
+  test-unit-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)